client/src/components/faqBlock/faqconfig.json

```diff
@@ -1,33 +1,31 @@
 {
     "General": {
         "Where and when is Aether happening?": "October 10 - 11th. The remote aspect will occur on the 10th and the in-person (on the 11th) will - tentatively - be at Donadeo Innovation Center for Engineering at the University of Alberta North Campus",
-        "How will the remote aspect work?": "TODO!!!",
```

Comment:
Are we going to update this?

Reply:
at a later date. I saved the one's i removed to a notepad for future reference